build.sbt

```diff
@@ -1,4 +1,4 @@
-ThisBuild / tlBaseVersion := "0.3"
+ThisBuild / tlBaseVersion := "0.2"
```

Comment:
We never released an `0.3` artifact and this PR works around the bincompat issues that led us to bump this in an earlier PR.